.pipelines/cni/cilium/cilium-cni-load-test.yaml

```diff
@@ -0,0 +1,120 @@
+
+stages:
+  - stage: creating_aks_cluster
+    displayName: "Create AKS Cluster with Cilium"
+    jobs:
+      - job: create_aks_cluster_with_cilium
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                make -C ./hack/swift azcfg AZCLI=az REGION=$(LOCATION)
+                make -C ./hack/swift overlay-cilium-up AZCLI=az REGION=$(LOCATION) SUB=$(SUBSCRIPTION_ID) CLUSTER=${RESOURCE_GROUP} NODE_COUNT=10 VM_SIZE=Standard_DS4_v2
+            name: "CreateAksCluster"
+            displayName: "Create AKS Cluster"
+  - stage: pod_deployment
+    dependsOn: creating_aks_cluster
+    displayName: "Pod Deployment"
+    jobs:
+      - job: deploy_pods
+        steps:
+          - task: AzureCLI@1
+            displayName: "Pod Deployment"
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                az extension add --name aks-preview
+                make -C ./hack/swift set-kubeconf AZCLI=az CLUSTER=${RESOURCE_GROUP}
+                bash hack/scripts/scale_deployment.sh
+  - stage: validate_state
+    dependsOn: pod_deployment
+    displayName: "Validate State"
+    jobs:
+      - job: validate_state
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                make -C ./hack/swift set-kubeconf AZCLI=az CLUSTER=${RESOURCE_GROUP}
+                kubectl get pods -A
+                bash hack/scripts/validate_state.sh
+            name: "ValidateState"
+            displayName: "Validate State"
+            retryCountOnTaskFailure: 3
+  - stage: connectivity_tests
+    dependsOn: validate_state
+    displayName: "Connectivity Tests"
+    jobs:
+      - job: cni_tests
+        steps:
+          - script: |
+              echo "install cilium CLI"
+              CILIUM_CLI_VERSION=$(curl -s https://raw.githubusercontent.com/cilium/cilium-cli/master/stable.txt)
+              CLI_ARCH=amd64
+              if [ "$(uname -m)" = "aarch64" ]; then CLI_ARCH=arm64; fi
+              curl -L --fail --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${CILIUM_CLI_VERSION}/cilium-linux-${CLI_ARCH}.tar.gz{,.sha256sum}
+              sha256sum --check cilium-linux-${CLI_ARCH}.tar.gz.sha256sum
+              sudo tar xzvfC cilium-linux-${CLI_ARCH}.tar.gz /usr/local/bin
+              rm cilium-linux-${CLI_ARCH}.tar.gz{,.sha256sum}

```

Comment:
their script needs to be flexible, but we know what arch and version we want - fix all of these variables so that we get the same thing every time

Reply:
You mean hardcode the cli version and arch version ?

Reply:
yes, we should pin those so that the CI is consistent